code/src/Components/CharCount.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const CharCount = ({ charCount }) => {
+  return (
+    <>
+      <svg
+        className="counter"
+        viewBox="0 0 31.83 31.83"
+        strokeDasharray={`${(charCount / 140) * 100} 100`}
+      ></svg>
+      <p className="message-count">
+        <span className={charCount > 140 ? "p-red" : "p-blue"}>
+          {140 - charCount}
+        </span>{" "}
+        /140
+      </p>
```

Comment:
Nice counter! Im not sure here thought, but the "p-red" "p-blue", were they supposed to change the color because I don't see any color change... it's in the className= so I guess it just changes the className in the span in some way.